travis/test_wordcount.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash
-
-# This script runs WordCount example locally in a few different ways.
```

Comment:
Not sure I can see it.

"Specifically, all combinations of: ..." doesn't appear, as far as I see.